geps/gep-1897/index.md

```diff
@@ -149,61 +117,53 @@ certificate, we implement a metaresource named `BackendTLSPolicy`, that was prev
-[GEP-713](../gep-713/index.md), that the implementation
-should watch the connections to the specified TargetRefs (Services), and if a Service matches a BackendTLSPolicy, then
-assume the connection is TLS, and verify that the TargetRef’s certificate can be validated by the client (Gateway) using
-the provided certificates and hostname before the connection is made. On the question of how to signal
-that there was a failure in the certificate validation, this is left up to the implementation to return a response error
-that is appropriate, such as one of the HTTP error codes: 400 (Bad Request), 401 (Unauthorized), 403 (Forbidden), or
+- How to verify the peer certificate, including signing certificate verification and other properties of the certificate (subject alt names).
+  - Support for referencing a CA certificate, as well as a help to use the default trusted system certificates.
+
+Two serve these dual roles, `BackendTLSPolicy` is usable in two contexts:
+* As a Direct Policy Attachment, `BackendTLSPolicy` can be applied to a Service in the same namespace.
+* As a Route backend ExtensionRef filter, a `BackendTLSPolicy` can be applied to override the service-attached policy, if any. This must exist in the same namespace as the Route (note: the extensionRef does not allow cross-namespace references, anyways).
+
+If a selected backend has both a service-attached and a backend filter, the backend filter takes precedence.
+As TLS is originated by the client, the ultimate decision in how that is done must also lie with the client.
+
+Failures during the TLS handshake may be handled with implementation-specific error codes, such as 500 or 503, or
 other signal that makes the failure sufficiently clear to the requester without revealing too much about the transaction,
 based on established security requirements.
 
-All policy resources must include `TargetRefs` with the fields specified
-in [PolicyTargetReference](https://github.com/kubernetes-sigs/gateway-api/blob/a33a934af9ec6997b34fd9b00d2ecd13d143e48b/apis/v1alpha2/policy_types.go#L24-L41).
-In an upcoming [extension](https://github.com/kubernetes-sigs/gateway-api/issues/2147) to TargetRefs, policy resources
-_may_ also choose to include `SectionName` and/or `Port` in the target reference following the same mechanics as `ParentRef`.
+BackendTLSPolicySpec contains the `Mode`, `TargetRefs` and `Validation` fields.
+
+* The `Validation` field is a `BackendTLSPolicyValidation` and contains `CACertificateRefs`, `WellKnownCACertificates`, and `Hostname`.
+* The `TargetRefs` is optional. It must be set when providing defaults for a Service, and must not be set when providing route level overrides.
+* The `Mode` field allows configuring the TLS mode. To start with, this will include `None` and `TLS`, though future versions will likely include a `MutualTLS` option. The `None` mode is useful when overriding on the route level; it is rejected if `TargetRefs` is ste.
```

Comment:
```suggestion
* The `Mode` field allows configuring the TLS mode. To start with, this will include `None` and `TLS`, though future versions will likely include a `MutualTLS` option. The `None` mode is useful when overriding on the route level; it is rejected if `TargetRefs` is set.
```